Formula/hydra.rb

```diff
@@ -27,9 +28,10 @@ def install
     inreplace "configure" do |s|
       # Link against our OpenSSL
       # https://github.com/vanhauser-thc/thc-hydra/issues/80
-      s.gsub! "/opt/local/lib", Formula["openssl@1.1"].opt_lib
+      s.gsub! "/opt/local/lib", "#{Formula["openssl@1.1"].opt_lib} #{HOMEBREW_PREFIX}/lib"
       s.gsub! "/opt/local/*ssl", Formula["openssl@1.1"].opt_lib
       s.gsub! "/opt/*ssl/include", Formula["openssl@1.1"].opt_include
+      s.gsub! "/opt/local/include", HOMEBREW_PREFIX/"include"
```

Comment:
https://github.com/vanhauser-thc/thc-hydra/blob/v9.2/configure isn't there `--with-ssl` options for this now?

Reply:
There should be, but `configure` promptly ignores `--with-ssl`: https://github.com/vanhauser-thc/thc-hydra/blob/fe930f4dd17d2949995499ce702e3b48946f7cbb/configure#L82-L128

The flag seems to have originated from an incomplete patch at https://github.com/vanhauser-thc/thc-hydra/issues/80.

Reply:
> I had a quick look at this earlier, and I was thinking it might be better to get rid of all of these and just `inreplace` the variables defined to be empty strings close to the start of the `configure` script. 

Yeah, that sounds neater. Changing course...